README.md

```diff
@@ -82,6 +82,7 @@ exec:exec@wait-for-container failsafe:integration-test
 exec:exec@stop-jfr-datasource exec:exec@stop-grafana exec:exec@stop-container
 exec:exec@destroy-pod`
 * or `bash repeated-integration-tests.sh 1`.
+* To run a single integration test without rebuilding, add the flag `-Dit.test=NameOfClassIT` in `STARTFLAGS=(...)` to `repeated-integration-tests.sh`, then run `bash repeated-integration-tests.sh 1`. You can also specify a comma-separated list of itests, e.g. `-Dit.test=AutoRulesIT,RecordingWorkflowIT`. Note that modifying an itest file does not require a rebuild.
```

Comment:
It's even been improved since you took your notes :-)

https://github.com/cryostatio/cryostat/blob/d5b3decef07d1ce326a4fd14a7e0dc2ed5d5d75c/repeated-integration-tests.bash#L43

So `bash repeated-integration-tests.bash 1 AutoRulesIT,RecordingWorkflowIT` would be equivalent and not require modifying the script file.

Reply:
Ah fantastic :) I can update the READme accordingly